doc/theoretical_description_regression.rst

```diff
@@ -29,11 +29,11 @@ feature vector :math:`X_{n+1}` such that
 1. The "Naive" method
 =====================
 
-The so-called naive method computes the residuals of the training data to estimate the 
+The so-called naive method computes the conformity scores of the training data to estimate the 
```

Comment:
I would keep the term "residuals" in the description of the methods as they are called like this in the papers.

Reply:
Done